weaviate/collections/batch/base.py

```diff
@@ -293,32 +290,49 @@ def failed_objects(self) -> List[ErrorObject]:
         """
         Get all failed objects from the batch manager.
 
+        Returns a copy so that the results can be used even if a new batch manager is opened.
+
         Returns:
             `List[ErrorObject]`
                 A list of all the failed objects from the batch.
         """
-        return self.__failed_objects
+        return copy(self.__failed_objects)
 
     def failed_references(self) -> List[ErrorReference]:
         """
         Get all failed references from the batch manager.
 
+        Returns a copy so that the results can be used even if a new batch manager is opened.
+
         Returns:
             `List[ErrorReference]`
                 A list of all the failed references from the batch.
         """
-        return self.__failed_references
+        return copy(self.__failed_references)
+
+    @property
+    def results(self) -> BatchResult:
+        """
+        Get the results of the batch operation.
+
+        NOTE: If you re-enter the batch manager using `with client.batch as batch` then the results will be reset.
 
-    def start(self) -> "_Batch":
+        Returns:
+            `BatchResult`
+                The results of the batch operation.
+        """
+        return self.__result
```

Comment:
not sure if this is a real proble, but might be confusing that some things return a copy and others a reference

Reply:
Ah yes, missed this, it should be a copy

Reply:
Oh no, actually not. `.results` is a property while `failed_x` are functions so I think it makes sense that the functions copy (perform an action) while the property just returns what exists. WDYT?

Reply:
hmmm - why is one a property and the other a function?

Not sure if the distinction is obvious to the average user

Reply:
For this very reason. If it's a potential confusion I will make `results()` a function that copies too

Reply:
👍 - I'm not sure if it is common to use properties for references and functions for copies